src/main/java/com/denizenscript/denizencore/scripts/commands/core/DebugCommand.java

```diff
@@ -33,19 +37,17 @@ public DebugCommand() {
     // Use to quickly output debug information to console.
     //
     // Valid types include:
-    // DEBUG: standard hideable debug.
+    // DEBUG: standard hideable debug, supports <@link language Script Logging Formats> (in which case it isn't hideable).

```

Comment:
If it's not standard hideable debug, it should be a separate type. ScriptDebug or something, idk.